src/App.tsx

```diff
@@ -1,26 +1,22 @@
-import React from 'react';
+import React, {useState} from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ModalConfirmCode from './components/ModalConfirmCode';
+import './components/ModalConfirmCode'
 
 function App() {
+  const [modalActive, setModalActive] = useState<boolean>(false);
```

Comment:
It is better to name with 'is' as it is boolean:

modalIsActive, setModalIsActive

2. No need to write boolean type here, as the TS can infer type by its meaning:

const [modalActive, setModalActive] = useState(false);

Reply:
done